RecoHI/HiTracking/python/HIBestVertexSequences_cff.py

```diff
@@ -19,18 +19,18 @@
 
 from RecoHI.HiTracking.HIPixelAdaptiveVertex_cfi import *
 hiOfflinePrimaryVertices=hiPixelAdaptiveVertex.clone( # vertexing run AFTER tracking
-    TrackLabel = cms.InputTag("hiGeneralTracks"),
+    TrackLabel = "hiGeneralTracks",
                                        
-    TkFilterParameters = cms.PSet(
-        algorithm = cms.string('filterWithThreshold'),
-        maxNormalizedChi2 = cms.double(5.0),
-        minPixelLayersWithHits=cms.int32(3),    #0 missing pix hit (Run 1 pixels)
-        minSiliconLayersWithHits = cms.int32(5),#at least 8 (3pix+5strip) hits total
-        maxD0Significance = cms.double(3.0),    #default is 5.0 in pp; 3.0 here suppresses split vtxs
-        minPt = cms.double(0.0),
-        maxEta = cms.double(100.),               
-        trackQuality = cms.string("any"),
-        numTracksThreshold = cms.int32(2)
+    TkFilterParameters = dict(
```

Comment:
This is perhaps correct, but for safety (not all parameters are re-defined here) I'd revert it to a `PSet`, as before